superdesk/metadata/item.py

```diff
@@ -524,11 +524,11 @@
         'type': 'boolean'
     },
 
-    # draft-js editor state
-    'editor_state': {
-        'type': 'list',
-        'nullable': True,
-        'mapping': not_enabled,
+    'fields_meta': {
+        'type': 'dict',
+        'mapping': {
+            'type': 'object',
```

Comment:
can you specify the properties? it would be better to avoid any dynamic mapping

Reply:
It can contain custom fields created by the user, so it's not possible to have a static mapping

Reply:
can you set it to `not_enabled` then? it shouldn't get indexed